pkg/antctl/antctl.go

```diff
@@ -175,6 +175,12 @@ $ antctl get podmulticaststats pod -n namespace`,
   $ antctl get networkpolicy
   Get the list of all control plane NetworkPolicies, sorted by the order in which the policies are evaluated.
   $ antctl get networkpolicy --sort-by=effectivePriority
+  Get the list of all control plane NetworkPolicies, sorted by the Name.
+  $ antctl get networkpolicy --sort-by=.sourceRef.name
+  Get the list of all control plane NetworkPolicies, sorted by the UID.
+  $ antctl get networkpolicy --sort-by=.metadata.uid
+  Get the list of all control plane NetworkPolicies, sorted by the CreationTimestamp.
+  $ antctl get networkpolicy --sort-by=.metadata.creationTimestamp
```

Comment:
Instead of 3 separate examples, I think we just need one, with a more generic comment mentioning the jsonPath (see `kubectl get --help` for the `--sort-by` flag)